tailcfg/tailcfg.go

```diff
@@ -486,7 +486,7 @@ type Node struct {
 	// This only applies to traffic originating from the current node to the
 	// peer or any of its subnets. Traffic originating from subnet routes will
 	// not be masqueraded (e.g. in case of --snat-subnet-routes).
-	SelfNodeV4MasqAddrForThisPeer *netip.Addr `json:",omitempty"`
+	SelfNodeV4MasqAddrForThisPeer *netip.Addr `json:",omitzero"`
```

Comment:
In the future, could this just be a `netip.Addr` (non-pointer)?

The `Addr` type has a convenient `IsValid` method, which I think is only ever false for the zero value.